Zend/zend_interfaces.c

```diff
@@ -549,9 +557,11 @@ const zend_function_entry zend_funcs_serializable[] = {
 	ZEND_FENTRY(unserialize, NULL, arginfo_serializable_serialize, ZEND_ACC_PUBLIC|ZEND_ACC_ABSTRACT|ZEND_ACC_CTOR)
 	ZEND_FE_END
 };
+
+const zend_function_entry *zend_funcs_throwable    = NULL;
 /* }}} */
 
-#define REGISTER_ITERATOR_INTERFACE(class_name, class_name_str) \
+#define REGISTER_INTERFACE(class_name, class_name_str) \
```

Comment:
Why I find this logical, don't forget that this will break some custom extensions that used the old macro.
I'm not against renaming it, but this must be documented into UPGRADING , and perhaps for 7.0, keep an alias macro for compat ?